horizons/gui/modules/singleplayermenu.py

```diff
@@ -140,6 +140,7 @@ def on_box_toggle(setting, setting_save_name):
 				box = self._gui.findChild(name=setting)
```

Comment:
This is not a nested definition. if --> def is ok.

Reply:
Could you please explain a little further, what a nested definition is then? I understood it as a function defined in another function (hence the nested). `on_box_toggle` adheres to this definition, but since you're telling me, that this is not a "nested definition", I don't seem to have full understanding of it. (Additionally, it is for --> def, not if --> def.)

Reply:
Of course is that a nested definition, but is not recognized with the pycodestyle check. I add a line.